### DISCUSSION TITLE
[Feature] UX. Efficient person selection with less clicks

### DISCUSSION THREAD

**Discussion by Micha10** [immich-app/immich#14187] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm giving every face a name. I have ~ 60.000 images and > 2000 persons. 
I would like to suggest some small improvements so the workflow will be better.

While viewing a picture, I click (1) on "edit people". Then I select the picture to edit (2). Since there are so many faces, I always wants to search the name, so I have to click (3) on the magnifying glass icon.

**My suggestion**: In that screen, immediately show the searchinput and set the focus/cursor in that control. Placeholder "type to search". 
So I can just click a face like before but I save one click to be able to search.

After saving the changes, I would like to go to the next picture (crsr right) and continue to change the persons, but I can't. After clicking on "edit", the tooltip "Info. Edited 3 people" is shown over the first 3 people. I always have to wait for the tooltip to disappear. 

I would like to **suggest** to display the tooltip at the lower right or at least down after the person-images, so the tooltip is out of the way.

Thank you for the great software

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#11235
